src/main/scala/org/holmesprocessing/totem/services/pemeta/pev.go

```diff
@@ -0,0 +1,575 @@
+package main
```

Comment:
if this is your code, I would recommend naming it PEMeta.go or something similar.